using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -2,68 +2,248 @@
 
 ## Events handled
 
-The following events are handled by the Identity extension:
+### Identity request identity
 
-### Hub shared state
+This event is used for the Identity extension in one of the following situations:
```

Comment:
please reword the end of this sentence to read "Identity extension to complete one of the following tasks:"